decentraland/unity-renderer
# Feat: player info area in new passport

## What does this PR change?

Fix #3313
Adds the player info area in the new passport

...

## How to test the changes?

1. Go to: https://play.decentraland.zone/?renderer-branch=feat/passport-info-section&ENABLE_new_avatar_flow=
2. Open the passport of a random user
3. Verify that the info are shown correctly in the top left area of the new passport as explained in the PRD and in the original issue #3313

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
